README.md

```diff
@@ -9,6 +9,8 @@ Running unit tests on anonymize.py using pytest
 pip install -r requirements.txt
 ## To run the API
 uvicorn app.api.main:app --reload  
-### To anonymize a file run the following command 
-In windows: **curl.exe -X POST -F "file=@tests\example.pdf" http://localhost:8000/anonymize**  
-The equivalent command for linux/mac: **curl http://localhost:8000/anonymize -F "file=@tests/example.pdf"**
+## Commands To Use The API  
+- ### /anomymize endpoint  
+  curl http://localhost:8000/anonymize -F "file=@tests/example.pdf"        
+- ### /ask endpoint  
+  curl "http://127.0.0.1:8000/ask?question=your_question"    
```

Comment:
`your-question` instead of `your question`. there's no whitespace in a url

Reply:
it's written your_question

Reply:
ok my bad, the whole url is underlined